### DISCUSSION TITLE
Adding Vietnamese language support for Spacy

### DISCUSSION THREAD

**Discussion by Avi197** [explosion/spaCy#9396] [Language Support]
Can I use a trained SoTA result Vietnamese language model to create a dataset and manually check for error for spacy pipelines? Or Spacy need a public dataset to properly implement a new language?
I'm planning on adding test, word vector and POS, NER for Vietnamese

**Comment by adrianeboyd** [comment]
Let's refer to the discussion here: https://github.com/explosion/spaCy/discussions/3056#discussioncomment-1442045

Additional tests in `spacy/tests/lang/__` for a particular language are always welcome, as are example sentences and language-specific settings in `spacy/lang/__`.

For most pretrained pipelines, we train our own word vectors internally so that we know exactly which data sources were used and to make sure that the tokenization is exactly the same as in `spacy`. But it some cases it also makes sense to accept contributions from users for well-documented / suitable word vectors, and that's also fine as long as the license is appropriate (examples: Catalan, Japanese, Russian).
